website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -74,7 +74,7 @@ Create a new Custom Resource Definition (CRD) API with group `cache` version `v1
 When prompted, enter yes `y` for creating both the resource and controller.
 
 ```console
-$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --resource --controller
+$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --plugins="deploy-image/v1-alpha" --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,modern,-v" --run-as-user="1001"
```

Comment:
@NikhilSharmaWe we need to revert this one because in the tutorial we are giving the steps to implement the code.

However, wdyt about we can add a note here such as : 

**Note:** You will check that in this tutorial we will be providing all steps to let you know how to implement the project. However, as a follow up you might want to check that would be possible to have the whole code generated to deploy and manage an Operand(image) using the [Deploy Image plugin](add a link for the kubebuilder doc) with the command  `$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --plugins="deploy-image/v1-alpha" --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,modern,-v" --run-as-user="1001"`